sqlmesh/core/model/decorator.py

```diff
@@ -43,7 +59,7 @@ def __init__(self, name: str, is_sql: bool = False, **kwargs: t.Any) -> None:
                         f"""Python model "{name}"'s `kind` dictionary must contain a `name` key with a valid ModelKindName enum value."""
                     )
 
-        self.name = name
+        self.name = name or ""
```

Comment:
Should we comment that this is temporarily set to `""` to allow the `__call__` method to infer the name based on the model's path?